nextflow.config

```diff
@@ -23,10 +23,10 @@ params {
     fasta             = null
     gtf               = null
 
-    // alevin-fry parameters (simpleaf)
-    simpleaf_rlen     = 91
+    // simpleaf parameters
```

Comment:
is the `simpleaf_rlen` not used/required anymore?
I remember we added once because somewhere it was necessary to add. Is this now defaulted in the tool?.

Reply:
I see. Super. Thanks for clarifying.